contents/docs/product-analytics/funnels.mdx

```diff
@@ -172,18 +172,30 @@ In the last touchpoint mode, one person will always only have one value.
 
 ##### 3. All steps
 
-In this case, the property value must be seen in all steps to be considered in the funnel.
+In this case, all events completed sequentially will be included in the funnel.
+
+Consider the following entirely possible example:
+
+1. `$pageview` with `browser = Chrome`
+2. `$pageview` with `browser = Firefox`
+3. `user signed up` with `browser = Chrome`
+4. `user posted` with `browser = Chrome`
+5. `user posted` with `browser = Firefox`
+
+With all steps mode, the person will be counted twice in the funnel, but in slightly different ways.
+
+The person will be in the Chrome breakdown as having completed all three steps: `$pageview`, `user signed up`, and `user posted`. The person will also be in the Firefox breakdown, but only for the `$pageview` step because they didn't complete the `user posted` step with Firefox.
```

Comment:
```suggestion
The person will be in the Chrome breakdown as having completed all three steps: `$pageview`, `user signed up`, and `user posted`. The person will also be in the Firefox breakdown, but only for the `$pageview` step because they didn't complete the `user signed up` step with Firefox.
```